app/questionnaire/questionnaire_schema.py

```diff
@@ -497,3 +497,8 @@ def _get_error_messages(self):
             messages.update(self.json["messages"])
 
         return messages
+
+    def get_individual_response_individual_section_id(self):
+        return self._questionnaire_json.get("individual_response", {}).get(
+            "individual_section_id"
+        )
```

Comment:
Would be nice to group this with the other IR methods in this class for clarity.

Reply:
Moved it to ir group now.